pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -409,6 +418,107 @@ func (r *Client) cacert() []byte {
 	return nil
 }
 
-func (c *Client) Finalize(vms []*planapi.VMStatus, planName string) {
+func (r Client) Finalize(vms []*planapi.VMStatus, planName string) {
```

Comment:
it also applies to openstack I guess, but the fact that we don't mark the status of the removal means that we may have leftovers when the controller restarts in the middle, right?

Reply:
yeah, this is kind of a best effort operation, we can however add a field like `finailizeCompleted: false` or something similar and have the controller rerun this, but I'm not sure about the risk/reward ratio here 😅

Reply:
yeah, me neither - after all, the goal is to move the workload to the target cluster, the workloads on the source should not be used after the migration.. just for backup